packages/@ember/-internals/glimmer/lib/helpers/fn.ts

```diff
@@ -6,6 +6,52 @@ import { InternalHelperReference } from '../utils/references';
 import buildUntouchableThis from '../utils/untouchable-this';
 
 const context = buildUntouchableThis('`fn` helper');
+
+/**
+@module ember
+*/
+
+/**
+  The `fn` helper is used to wrap the provided callback function up with a
```

Comment:
I think this could use a more generic, new-learner explanation of what to use it for. Normally I'd say that API docs are fine to be more technical, but this one is the "new way to do actions" and will therefore be higher traffic for new learners. Here's an example:

_The `fn` helper lets you call a function in your template and pass it some arguments. For example, someone might use it to register a click handler:_
_(show example)_
_The way the `fn` helper works is, it wraps the provided callback callback function up with a..._

Reply:
@jenweber I tweaked the docs significantly, mind re-reviewing?